apps/Doom 3/uninstall

```diff
@@ -1,12 +1,4 @@
 #!/bin/bash
 
 purge_packages || error "Dependencies failed to uninstall"
-if [ -d ~/RPIDoom3Installer ];then
-  cd ~/RPIDoom3Installer || "Failed to Change directory"
-  echo "Running uninstall script..."
-  ./uninstall.sh || error "Failed to execute uninstall script :("
-else
-  echo "WARNING: Doing nothing as ~/RPIDoom3Installer folder does not exist!"
-fi
-
-exit 0
+sudo rm -rf ~/RPIDoom3Installer ~/Doom3GameFiles ~/Doom3Demo ~/Desktop/Doom3Demo.desktop ~/.local/share/applications/Doom3Demo.desktop ~/.local/share/applications/Doom3.desktop
```

Comment:
During an update reinstall, it's probably best to avoid deleting the user's gamefiles.

Reply:
I will note that this can't be helped now. because this is how the previous script did it. I can change that for the future but I believe on update, the old uninstall script is run.
is that correct?

Reply:
ignore what I said. saves aren't stored in these folders. they reside in

``` "%s/.local/share/dhewm3", getenv("HOME"));```